runtime/v2/logging/logging_unix.go

```diff
@@ -0,0 +1,52 @@
+// +build !windows
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package logging
+
+import (
+	"context"
+	"os"
+	"os/signal"
+
+	"golang.org/x/sys/unix"
+)
+
+// Run the logging driver
+func Run(fn LoggerFunc) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	config := &Config{
+		ID:        os.Getenv("CONTAINER_ID"),
+		Namespace: os.Getenv("CONTAINER_NAMESPACE"),
+		Stdout:    os.NewFile(3, "CONTAINER_STDOUT"),
+		Stderr:    os.NewFile(4, "CONTAINER_STDERR"),
+	}
+	var (
+		sigCh = make(chan os.Signal, 32)
```

Comment:
Might be eluding me, but any reason for this buffer size?

Reply:
not sure, but my guess would to maybe cover all possible linux signals https://www.man7.org/linux/man-pages/man7/signal.7.html?

Reply:
We're only signed up to get notified for SIGTERM though is my confusion.

Reply:
yeah, I can't really give a definitive answer here, since the original implementation had it and I wanted to leave it as is. the change in windows counterpart makes sense.